docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -42,15 +43,19 @@ A formula should be disabled to indicate to users that the formula cannot be use
 
 The most common reasons for disabling a formula are:
 
-- it cannot be built from source (meaning no bottles can be built)
+- it cannot be built from source (meaning no new bottles can be built)
 - it has been deprecated for a long time
 - the upstream repository has been removed
 - the project has no license
 
-Formulae should not be disabled without a deprecation period of at least three months unless the circumstances are exceptional (e.g. the formula does not build on any supported macOS version or Linux). Popular formulae should have longer deprecation periods. The popularity of a formula should be based on our analytics data.
+Popular formulae (i.e. have more than 999 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/install/30d/)) should be deprecated for at least six months even if e.g. they do not build from source and do not have a license before being disabled.
+
+Unpopular formulae (i.e. have fewer than 1000 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/install/30d/)) can be deprecated immediately for any of the reasons above e.g. they cannot be built from source on any supported macOS version or Linux.
```

Comment:
Everything can be deprecated immediately - does this mean disabled immediately?

Reply:
@Bo98 I don't think so but I'm open to opinions to the contrary.

Reply:
It reads to me with the same conflict as Bo

1. Popular formula > Deprecate > wait 6 months > Disable
2. Unpopular formula > Deprecate > then what?

It's unclear how these would be handled differently, so I would assume unpopular formula should actually be "disabled" immediately.

Reply:
Ah, reread, yeh, this should be disabled I think.